openapi/openapi.yml

```diff
@@ -1116,6 +1116,9 @@ paths:
                 storage_size:
                   description: Requested storage size in GiB
                   type: integer
+                version:
+                  description: PostgreSQL version
+                  type: string
```

Comment:
Huh so no tests submitted this key? Because they'd break.

Reply:
From the output `git grep version spec/routes/api`, looks like the cli pg create tests were the first to use the parameter.